state-chain/runtime/src/lib.rs

```diff
@@ -1342,6 +1344,53 @@ impl_runtime_apis! {
 				None
 			}
 		}
+
+		fn cf_witness_count(hash: pallet_cf_witnesser::CallHash) -> Option<FailingWitnessValidators> {
```

Comment:
I think it would be useful to give a full list of who did and didn't vote instead of just the missing votes. Basically a Vec of { id, vanity, vote }.